app/public/templates/landing_page.html

```diff
@@ -11,7 +11,8 @@ <h1>{{ scenario_name }}</h1>
     </button>
     <script type="text/javascript">
       function playGame() {
-        let nextURL = "/{{ scenario_friendly_name }}/0";
+        let nextURL = "/{{ scenario_friendly_name }}/tutorial";
+        alert("You are entering the tutorial.");
```

Comment:
I think we can remove this. Any reason we want to alert? You mentioned in the loom video that you intentionally left it in, but I don't see the use case. Please let me know :)

Reply:
I did not know how the modal will look like so I added that alert as a placeholder for the modal.

Should I remove it?

Reply:
Oh we have a modal prior to going into tutorial? I see. Okay we can leave that then